common/v2/features/Dashboard/components/WalletBreakdown/WalletBreakdownView.tsx

```diff
@@ -18,13 +18,29 @@ const { BRIGHT_SKY_BLUE } = COLORS;
 const { SCREEN_MD } = BREAK_POINTS;
 
 const BreakDownHeading = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: baseline;
   margin: 0;
   font-size: 20px;
   font-weight: bold;
   color: #424242;
 
   @media (min-width: ${SCREEN_MD}) {
     font-size: 24px;
+    flex-direction: row;
```

Comment:
The designs show the label inline with the title even on mobile.
Since the figma mobile design is obviously broken and we still have space on medium screens, lets display inline as soon as possible (ie. `SCREEN_SM`)

Reply:
Moved inline display when screen > SCREEN_XS